pubspec.yaml

```
@@ -9,12 +9,12 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.17.0
+  intl: ^0.17.0-nullsafety.2
```

```suggestion
  intl: ^0.17.0
```

This version is already null safe

---

Using `^0.17.0` gives the pub error:

```
Because every version of flutter_test from sdk depends on path 1.8.0-nullsafety.3 and intl >=0.17.0 depends on path ^1.8.0, flutter_test from sdk is incompatible with intl >=0.17.0.

So, because flutter_localized_locales depends on both intl ^0.17.0 and flutter_test any from sdk, version solving failed.
```